py-scripts/lf_report.py

```diff
@@ -503,26 +499,26 @@ def update_allure_results_history(self,allure_results_path=""):
         try:
             files = os.listdir(self.allure_report_history)
             for fname in files:
-                allure_report_history_file =  str(self.allure_report_history_path) + '/' + str(fname)
+                allure_report_history_file = str(self.allure_report_history_path) + '/' + str(fname)
                 allure_results_history_file = str(self.allure_results_history_path) + '/' + str(fname)
-                shutil.copy(allure_report_history_file,allure_results_history_file)
+                shutil.copy(allure_report_history_file, allure_results_history_file)
 
         except Exception as x:
             traceback.print_exception(Exception, x, x.__traceback__, chain=True)
             logger.info("Either no allure report present or the copy of history failed.")
 
     def copy_allure_report(self):
         # TODO abiltiy to set the Allure results dir
-        if allure_results == "":
-            self.allure_results_history_path = os.path.join(self.path_date_time,"history")
+        if allure_results == "":                                                                            # noqa F821 - undefined name
+            self.allure_results_history_path = os.path.join(self.path_date_time, "history")
             self.allure_results = "{allure_results_path}".format(allure_results_path=self.path_date_time)
 
         else:
-            #TODO check if the path is passed in for allure results
-            self.allure_results = allure_results
-            self.allure_results_history_path = os.path.join(self.allure_results,"history")
+            # TODO check if the path is passed in for allure results
+            self.allure_results = allure_results                                                            # noqa F821 - undefined name
```

Comment:
Does this pass w/ all the whitespace after the conditional? If so, let's just get it to two spaces after, as flake checks for minimum of